db/migrate/20230815200321_add_ref_users_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddRefUsersToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, null: false, foreign_key: true
```

Comment:
- [ ] The `foreign_key` attribute should refer to the users, even if the column id would be `author_id`

  ```
  ...foreign_key: { to_table: :users }
  ```

- [ ] After fixing these issues you still need to migrate your database schema and set up the `db/schema.rb` file, which is absent currently. Kindly run the command below.

  ```
  rails db:migrate
  ```